src/Sulu/Bundle/TagBundle/Controller/TagController.php

```diff
@@ -356,4 +357,40 @@ protected function getData(Request $request)
     {
         return $request->request->all();
     }
+
+    /**
+     * Returns array of tags from escaped input string.
+     */
+    private function explodeTags(string $input): array
```

Comment:
I have a very hard time understanding this function :confused: Isn't there an easier solution to this problem? Can't we escape the comma in a different way?

Reply:
The problem is, that the escape character could be escaped as well, otherwise i could explode by `/(?<!\\),/`, but since i cannot make a negative lookbehind, which checks that the comma is not preceded by an odd amount of backslashes, it's not possible with regex.